home/src/app/services/login/login.service.js

```diff
@@ -86,7 +86,11 @@
             sessionStorage.removeItem('userProfile');            
             sessionStorage.removeItem('accessToken');            
             $http.defaults.headers.common['Authorization'] = undefined;
-            delete $rootScope.userProfile;       
+            delete $rootScope.userProfile;
```

Comment:
window reload ?. I don't think it is a good idea
Why you don't use $location.path instead ?